### DISCUSSION TITLE
Custom Modernization Rule

### DISCUSSION THREAD

**Discussion by deepthiqantas** [adobe/aem-modernize-tools#206] [Q&A]
Hi,

I am trying to customise page template modernization rule and implemented StructureRewriteRule. Have updated the config values I created for this custom service. But the tool doesnt not pick up these rules.

Is there anything I am missing here?

**Comment by bstopp** [comment]
Can you provide your code or a reference to it? My guess is you're missing the `@Component` annotation on your custom Rule.
